docs/release/3.4/release-3-4-44.md

```diff
@@ -0,0 +1,235 @@
+OSG Software Release 3.4.44
+===========================
+
+**Release Date**: 2020-02-20    
+**Supported OS Versions:** EL7, EL6
+
+!!!tip "Want faster access to production-ready software?"
+    OSG 3.5 and 3.4 offer a rolling release repository where packages are added as soon as they pass acceptance testing.
+    To install packages from this repository, enable `[osg-rolling]` in `/etc/yum.repos.d/osg-rolling.repo`:
+
+        [osg-rolling]
+        ...
+        enabled=1
+
+    Or for one-time installations, append the following to your `yum` command:
+
+        --enablerepo=osg-rolling
+
+Summary of changes
+------------------
+
+This release contains:
+
+-   [XRootD 4.11.2](https://github.com/xrootd/xrootd/blob/v4.11.2/docs/ReleaseNotes.txt): Bug fix release
+-   XCache 1.2.1: Fixed problem where plugins were applied to redirectors
+-   [VO Package v99](https://github.com/opensciencegrid/osg-vo-config/releases/tag/release-99): New certificate DN for HCC VOMS
+-   UberFTP 2.8-3: OSG fixes incorporated upstream
+-   osg-configure 2.5.1: Minor fixes
+-   osg-system-profiler 1.5.0:
+    -   Report XRootD configuration
+    -   Elide kernel RPM checks
+
+These [JIRA tickets](https://jira.opensciencegrid.org/issues/?jql=project%20%3D%20SOFTWARE%20AND%20fixVersion%20%3D%203.4.44%20ORDER%20BY%20priority%20DESC%2C%20key%20DESC) were addressed in this release.
+
+Containers
+----------
+
+The StashCache container is available and has been tagged as `stable` in accordance with our
+[Container Release Policy](https://opensciencegrid.org/technology/policy/container-release/)
+
+-   [StashCache](https://hub.docker.com/r/opensciencegrid/stash-cache/)
+
+Notes
+-----
+
+This section describes important upgrade notes and/or caveats for packages available in the OSG release repositories.
+Detailed changes are below. All of the documentation can be found [here](/index.md).
+
+-   OSG 3.4 contains only 64-bit components.
+-   The StashCache service is only supported on EL7
+
+### GlideinWMS ###
```

Comment:
I think we can remove this section. I don't believe we have any GlideinWMS hosts on these old versions anymore

Reply:
Sounds good to me.